### PR TITLE
adding a round history

### PR DESCRIPTION
**Pull request by florianklockenkemper**
added a history for played rounds including a new scoreboard

### CODE REVIEW

**Comment by mitelg** [comment]
Hey @florianklockenkemper 
thanks for your contribution.
One thing: AppBundle\Entity\Participant and AppBundle\Entity\Round are missing

File: src/AppBundle/Controller/DokoController.php [hunk 1]
```diff
@@ -114,16 +102,29 @@ public function enterPointsAction(Request $request)
...
             }
 
+            $round = new Round();
+            $round->setBock($pointsForm->get('bockRound')->getData());
+            $round->setCreationDate(new DateTime());
+
             // if data is okay, save points to database
+            $participants = new ArrayCollection();
             foreach ($data as $item) {
+                $points = $item['points'];
```
Comment (mitelg) [inline src/AppBundle/Controller/DokoController.php]: calculateGameResult() is already handling bock rounds.
or did you delete the Game entity?

**Comment by florianklockenkemper** [comment]
I added the missing files.

File: src/AppBundle/Controller/DokoController.php (hunk 1 above)
Comment (florianklockenkemper) [inline src/AppBundle/Controller/DokoController.php]: I didn't wanted to delete the game entity totally.
I just wanted that every round is marked as 'Bock' for the record.
How to save the number of left bockrounds should be improved at another place.

**Comment by mitelg** [comment]
@florianklockenkemper looks better now :wink: 
but is it possible, that the bootstrap files are loaded on the installation and are not part of the project? just similar to `composer install`

File: src/AppBundle/Controller/DokoController.php (hunk 1 above)
Comment (mitelg) [inline src/AppBundle/Controller/DokoController.php]: please delete the game entity and all its usages, as we don't need it anymore at the moment. If we would like to use it in the future, we could introduce it again

**Comment by mitelg** [comment]
thanks @florianklockenkemper !

**Pull request merged**